tests/unit/__init__.py

```diff
@@ -1,3 +1,8 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
+
 """Ops testing settings."""
+
+import ops.testing
+
+ops.testing.SIMULATE_CAN_CONNECT = True  # type: ignore
```

Comment:
Out of curiosity, what is the intention behind `SIMULATE_CAN_CONNECT`? I've removed in WordPress charm on this [commit](https://github.com/canonical/wordpress-k8s-operator/commit/93d21d5dd00c39ad38c6ff0fa8f2bf79ba16ad87) before, was wondering in which test this is required. :)

Reply:
Iirc it has been deprecated